src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java

```diff
@@ -0,0 +1,13 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import java.util.Date;
+
+public class RandomDataProvider {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1118)